filebeat/input/filestream/identifier.go

```diff
@@ -133,6 +144,15 @@ func (p *pathIdentifier) Name() string {
 	return p.name
 }
 
+func (p *pathIdentifier) Supports(f identifierFeature) bool {
+	switch f {
+	case trackRename:
+		return true
```

Comment:
shouldn't this return `false` and the `inodeDeviceIdentifier` return `true` instead?